dlp/dlp_snippets/inspect.go

```diff
@@ -315,12 +328,14 @@ func inspectDatastore(w io.Writer, client *dlp.Client, project string, minLikeli
 // [START dlp_inspect_bigquery]
 
 // inspectBigquery searches for the given info types in the given Bigquery dataset table.
-func inspectBigquery(w io.Writer, client *dlp.Client, project string, minLikelihood dlppb.Likelihood, maxFindings int32, includeQuote bool, infoTypes []string, pubSubTopic, pubSubSub, dataProject, datasetID, tableID string) {
+func inspectBigquery(w io.Writer, client *dlp.Client, project string, minLikelihood dlppb.Likelihood, maxFindings int32, includeQuote bool, infoTypes []string, customDictionaries []string, customRegexes []string, pubSubTopic, pubSubSub, dataProject, datasetID, tableID string) {
```

Comment:
Do all of these need to use custom? There are already so parameters here, I'm hesitant to add more.

Reply:
Custom info types can be used in all types of inspection, so I wanted to include them in all the samples. You could certainly argue that just putting them in inspectString and inspectFile is sufficient, and if you feel strongly I can remove them from these methods.

Reply:
Whichever is consistent with the other languages. Seems most natural to me to have a single `inspectCustom` sample with corresponding docs which explains how to do custom types.

Reply:
The other code samples use custom info types in all versions of inspect*, so if you want it to be consistent then the current structure is right.